package.json

```diff
@@ -37,7 +37,7 @@
         "aperture-config": "git+https://github.com/joyent/aperture-config.git#master",
         "joyent-schemas": "git+https://github.com/joyent/schemas.git#caf3a226ed0707f5da897e1da151cc6d97fccda2",
         "jsprim": "0.6.1",
-        "verror": "1.6.1"
+        "verror": "1.10.0"
```

Comment:
Just curious, why do we need a new version of verror? Is it to get the `hasCauseWithName` function?